website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 page_title: Policy Sets - API Docs - Terraform Cloud
 description: >-
-  Use the `/policy-sets` endpoint to manage groupings of Sentinel and OPA policies. List, show, create, and update policy sets and policy set versions, and attach and detach policies from workspaces and projects using the HTTP API.
+  Use the `/policy-sets` endpoint to manage groupings of Sentinel and OPA policies. List, show, create, and update policy sets and policy set versions, and attach and detach policies from workspaces, projects, and exclusions using the HTTP API.
```

Comment:
I wonder if there is a better way to word this whole thing 🤔 I'll think about this more! It's getting a bit unruly and finding it hard to understand. Rose might have some thoughts!

Reply:
@rkoron007